manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+  "manifest_version": 3,
+  "name": "Hide Amazon Cart",
+  "version": "0.0.1",
+  "description": "Hide the Amazon cart sidebar while browsing on Amazon",
+   "permissions": ["scripting"],
+  "icons": {
+    "16": "images/cart-16.png",
+    "32": "images/cart-32.png",
+    "48": "images/cart-48.png",
+    "128": "images/cart-128.png"
+},
+"background": {
+    "service_worker": "scripts/background.js"
+},
+"web_accessible_resources": [
+  {
+    "resources": ["css/*.css"],
+    "extension_ids": [
+      "jijclebhnnihgcgjabjgcdbapeccoodc"
+    ]
+    }
+  ],
+"content_scripts": [
+    {
+        "matches": [
+        "*://www.amazon.com/*"
```

Comment:
I thought about being more specific with the URLs but wasn't sure what the pattern was for Amazon item pages. The only other URL that needs this access is `/cart`, but that's included within `www.amazon.com`. Once I figure this out I'll update the URL to limit access.